### DISCUSSION TITLE
Integrate local 3d content as seamless as images

### DISCUSSION THREAD

**Discussion by mniehus** [squidfunk/mkdocs-material#3190] [Ideas]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I want to suggest an idea and checked that ...

- [X] ... to my best knowledge, my idea wouldn't break something for other users
- [X] ... the documentation does not mention anything about my idea
- [X] ... there are no open or closed issues that are related to my idea

### Description

Hi . iframe embedding 3d content from online sites like e.g. sketchfab.com works like a charm. Nevertheless,  a feature/method would be great that allow to integrate local 3d content as seamless as images. any py or js thoughts ?

### Use Cases

3d contents will be one of the outstanding features in next-gen sites

**Comment by squidfunk** [comment]
I'm not sure I understand what you mean. Since this is not specific to Material for MkDocs, but MkDocs in general, I'm converting this to a discussion. Could you provide some more details what you mean specifically?

**Reply by mniehus** [reply]
hi martin. of course yes. i would like to embed a 3d image as for example [here (user)](https://help.sketchfab.com/hc/en-us/articles/205611655-Using-Markdown) or [here (developer)](https://sketchfab.com/developers/viewer). In any case I wonder if there is any ongoing effort for example via plugin. I am not aware of any, but think it would be welcome. thanks and greetings.

**Reply by squidfunk** [reply]
Thanks for the input. I'm sure this could be pulled off with some extra JavaScript, CSS, and HTML, or a MkDocs plugin or Markdown extension that does the heavy lifting for the user. It's however beyond the scope of this project.
